### PR TITLE
Styling has been added to the BlogItem Component

### PR DESCRIPTION
**Pull request by zsubzwary**
One thing to remember is that I have inserted hard-coded date in post timestamps i.e `Posted on: March 20, 2019` for testing purpose !!

### CODE REVIEW

File: BlogApp/src/BlogItem.js [hunk 1]
```diff
@@ -1,22 +1,29 @@
...
+        </div>
       </div>
-      <div>Text: {blog.text}</div>
-      <div>
-        <button>Edit</button>
-        <button>Delete</button>
+      <div style={styles.blogItemMainText}> {blog.text} </div>
+      <div style={styles.blogItemFooter}>
+        <div style={styles.blogItemTimestamp}>
+          Posted on: March 20, 2019{/*blog.timestamp*/}
```
Comment (zsubzwary) [inline BlogApp/src/BlogItem.js]: I think the following should be replaced later on to get the real date of each blog post !! 😇

```suggestion
          Posted on: {blog.timestamp}
```

**Review by imrankhokhar — COMMENTED** [review]
Wouldn’t it be great if the hover effect was added on the buttons just like Bootstrap has ??

**Comment by zsubzwary** [comment]
Yes @imrankhokhar that thought also crossed my mind, but because I am not using CSS _(we are working on JSX)_ so it would become a little bit complicated....

That's why I didn’t made those changes !! 😇

**Review by imrankhokhar — APPROVED** [review]
Okay then !!

Let’s make it fly !! 🚀

**Review by ZohaibAhmad786 — APPROVED** [review]
Nice Design but you have to make it more stylish just like WhatsApp..

**Comment by zsubzwary** [comment]
Dear @ZohaibAhmad786 WhatsApp isn’t what we are building in here !!
This is a type of micro-blogging site so I think the design is OKAY for now !!

The new layout has somewhat similarity to the Blog
# 👇👇👇
Image Showing Modern Design

**Review by shassaan — COMMENTED** [review]
overall code looks fine to me I think we should use arrow function for dump components

**Comment by zsubzwary** [comment]
That is a nice suggestion @shassaan but the problem is that, if I did the change right now in here, then most of the students who haven't been following the PR conversation will just miss it out !!

I think we will request Sir @thebazshah to do it in class in-front of everyone !! 
Is it okay @shassaan ..??

**Comment by zsubzwary** [comment]
@shassaan @ZohaibAhmad786  @imrankhokhar 
Here is how it changed !! 
# 👇  👇  👇 👇  👇 😇 
An Animated GIF showing the changes !!

**Comment by shassaan** [comment]
yes its okay @zsubzwary

**Comment by shassaan** [comment]
yes its okay @zsubzwary

**Pull request merged**